lib/plugins.js

```diff
@@ -33,12 +33,20 @@ module.exports.load = function ({ config }) {
       }
 
       // register schema and validate settings
+      let validate;
       try {
-        schemas.register('plugin', pluginName, plugin.schema)(settings);
+        validate = schemas.register('plugin', pluginName, plugin.schema);
       } catch (e) {
         logger.error(`Wrong schema definition for ${pluginName}`, e);
       }
 
+      if (validate !== undefined) {
```

Comment:
Do not check for such value — I think it's a bad fallback and a situation that should never happen. I'd rather crash as soon as possible and checkout what's wrong instead of silently swallowing this exception.